### ISSUE TITLE
strong sign?

### ISSUE THREAD

**Issue by apobekiaris** [akarnokd/reactive-extensions#16]
is it possible to sign the library?

**Comment by akarnokd** [comment]
How does that work? I have no idea how to do that.

**Comment by apobekiaris** [comment]
hmm not sure what you mean? I need to consume the lib from strongly sign assemblies so it needs to be sign also 

```

System.IO.FileLoadException
Could not load file or assembly 'reactive-extensions, Version=0.0.26.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. A strongly-named assembly is required. (Exception from HRESULT: 0x80131044)
   at GitHubManager.Module.Services.NgrokService..cctor()
```

**Comment by apobekiaris** [comment]
>I have no idea how to do that.

really? the mighty akarnokd? hard to believe :).  

Just go to project properties Sign assembly tab and follow the wizard to create a new key

**Comment by apobekiaris** [comment]
here some command like if u prefer

https://www.c-sharpcorner.com/article/how-to-give-strong-name-to-dll-and-how-to-install-dll-in-g/

**Comment by akarnokd** [comment]
> really? the mighty akarnokd? hard to believe :).

Am I known for in-depth .NET ecosystem/infrastructure knowledge?

**Comment by apobekiaris** [comment]
i guessed judging from your contributions did not want to imply anything, apologies

**Comment by akarnokd** [comment]
I've released 0.0.27-alpha with supposedly strong-named assembly. Please let me know if it worked or not.

**Comment by apobekiaris** [comment]
works fine many thnks.
